assignment1.txt

```diff
@@ -0,0 +1,21 @@
+Julia Tan
+
+1. The 'return 0' at the end of the program is a convention that represents the success of the program. It is required for C programs.
+
+2. Instead of "%d", it should say "%f" because the variable 'x' is a float, not an integer.
```

Comment:
Correct! You're missing a couple of more errors though - see if you can find them!